<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TxtName</class>
 <widget class="QWidget" name="TxtName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>146</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string>请输入这个模板的名称</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="le_fileName"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pb_true_flie_name">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
